src/klassy.js

```diff
@@ -0,0 +1,12 @@
+import bem from 'react-bem-helper';
+
+export default function klassy (packageJSON, override) {
```

Comment:
Consider using https://github.com/JedWatson/classnames package, might be useful to get rid of boilerplate code.